settings/__init__.py

```diff
@@ -5,3 +5,14 @@
     from .local import *  # noqa
 except ImportError as error:
     raise ImportError("No local.py settings file found. Did you remember to add your local.py?")
+
+# raise an exception if the root domain is production.
+if '/osf.io' in OSF_HOME:
+    raise Exception(
+        'OSF UI tests should *never* be run against production. '
```

Comment:
Ultimately we are going to want to run this against Prod since that's its primary use case, but we can talk strategies for limiting problems later.